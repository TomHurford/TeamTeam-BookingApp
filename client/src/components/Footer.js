import React from "react";
import {Link} from 'react-router-dom';
function Footer(){
    return(
      <footer className="footer">
          <ul>
           <li><p><Link to = "/about">About</Link></p></li>
           <li><p><Link to = "/privacy">Privacy</Link></p></li>
           <li><p><Link to = "/terms">Terms</Link></p></li>
          </ul>
      </footer>
    )
}
export default Footer;
