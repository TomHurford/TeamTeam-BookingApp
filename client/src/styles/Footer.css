.footer{
    position: fixed;
    bottom: 0;
    left: 0;
    width: 100%;
    background-color: black;
    text-align: right;
    padding-top: 2px;
    padding-bottom: 2px;
}
