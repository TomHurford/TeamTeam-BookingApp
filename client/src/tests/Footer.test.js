import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';
import { BrowserRouter } from 'react-router-dom';
// Snapshot test for Footer
test('Footer snapshot test', () => {
  const component = renderer.create(
    <BrowserRouter>
      <Footer />
    </BrowserRouter>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});