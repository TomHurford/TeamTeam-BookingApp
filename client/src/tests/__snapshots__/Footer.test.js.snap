// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot test 1`] = `
<footer
  className="footer"
>
  <ul>
    <li>
      <p>
        <a
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
      </p>
    </li>
    <li>
      <p>
        <a
          href="/privacy"
          onClick={[Function]}
        >
          Privacy
        </a>
      </p>
    </li>
    <li>
      <p>
        <a
          href="/terms"
          onClick={[Function]}
        >
          Terms
        </a>
      </p>
    </li>
  </ul>
</footer>
`;
